parcels/particle.py

```diff
@@ -457,6 +473,94 @@ def show(self, **kwargs):
         plt.show()
         plt.pause(0.0001)
 
+    def show_velocity(self, **kwargs):
```

Comment:
Would it make sense to make this actually part of the original `pset.show()` function? There's quite a bit of overlap with that function, so for example handling of the time and title doesn't need to be replicated.